### PR TITLE
Add notes about more caveats.

### PR DESCRIPTION
**Pull request by jugmac00**
- add note about cmp function
- add note about Python 2 incompatible changes

### CODE REVIEW

**Comment by jugmac00** [comment]
This fixes #7

File: README.rst [hunk 1]
```diff
@@ -26,7 +26,13 @@ as otherwise the values will get lost.
 Caveats
 =======
 
-During rewrite double hyphens within HTML-comments are removed as the Chameleon
+- During rewrite double hyphens within HTML-comments are removed as the Chameleon
 engine in Zope 4 (and the `actual specification`_) is very strict about it.
```
Comment (icemac) [inline README.rst]:
```suggestion
   engine in Zope 4 (and the `actual specification`_) is very strict about it.
```

File: README.rst [hunk 2]
```diff
@@ -26,7 +26,13 @@ as otherwise the values will get lost.
...
 
+- The underlying lib2to3 does not take into account, that the `cmp` function
+is no longer available in Python 3.
```
Comment (icemac) [inline README.rst]:
```suggestion
  is no longer available in Python 3.
```

File: README.rst [hunk 3]
```diff
@@ -26,7 +26,13 @@ as otherwise the values will get lost.
...
+
+- This tool converts Python 2 to Python 3 - that means the code may not be
+compatible with Python 2 any more.
```
Comment (icemac) [inline README.rst]:
```suggestion
  compatible with Python 2 any more. For these edge cases manual changes are required to make it
  compatible with both versions.
```

**Comment by jugmac00** [comment]
Thanks for the review! I used the web GUI to apply your changes which created a separate commit for each fix - maybe better squash the commits.

**Comment by icemac** [comment]
Thank you!

**Pull request merged**